appeals/web/src/server/appeals/appeal-details/allocation-details/allocation-details.controller.js

```diff
@@ -257,11 +259,19 @@ export const postAllocationDetailsCheckAnswers = async (request, response) => {
 		delete request.session.allocationLevel;
 		delete request.session.allocationSpecialisms;
 
-		addNotificationBannerToSession({
-			session: request.session,
-			bannerDefinitionKey: 'allocationDetailsUpdated',
-			appealId: appealDetails.appealId
-		});
+		if (appealDetails.allocationDetails) {
```

Comment:
This could probably be shortened to ```addNotificationBannerToSession({
				session: request.session,
				bannerDefinitionKey: appealDetails.allocationDetails ? 'allocationDetailsUpdated' : 'allocationDetailsAdded',
				appealId: appealDetails.appealId
			});```